core/src/main/resources/schema/infinispan-config-10.0.xsd

```diff
@@ -266,6 +266,11 @@
         <xs:documentation>Indicates the default cache for this cache container</xs:documentation>
       </xs:annotation>
     </xs:attribute>
+    <xs:attribute name="zero-capacity-node" type="xs:boolean" default="false">
+      <xs:annotation>
+        <xs:documentation>If 'false' then no data is stored in this node. Defaults to 'false'.</xs:documentation>
```

Comment:
Maybe add a note mentioning that it is equivalent to setting capacity factor 0 on all clustered caches.

Reply:
It's not exactly true, because internal caches won't have capacity factor 0, and with this modification all caches are capacity factor 0

Reply:
do you mean `If true`?